doc/nrf/installation/install_ncs.rst

```diff
@@ -0,0 +1,599 @@
+
+         * **Branch**: To check out the ``main`` branch that includes the latest state of development, enter the following command:
+
+           .. code-block:: console
+
+              west init -m https://github.com/nrfconnect/sdk-nrf --mr main
+
+         This will clone the manifest repository `sdk-nrf`_ into :file:`nrf`.
+
+         Initializing west with a specific revision of the manifest file does not lock your repositories to this version.
+         Checking out a different branch or tag in the `sdk-nrf`_ repository and running ``west update`` changes the version of the |NCS| that you work with.
+
+         .. west-error-start
+
+         .. note::
+
+            If you get an error message when running west, :ref:`update west <west_update>` to the latest version.
+            See :ref:`zephyr:west-troubleshooting` in the Zephyr documentation for more information.
+
+         .. west-error-end
+
+      #. Enter the following command to clone the project repositories:
+
+         .. code-block:: console
+
+            west update
+
+         Depending on your connection, this might take some time.
+      #. Export a :ref:`Zephyr CMake package <zephyr:cmake_pkg>`.
+         This allows CMake to automatically load the boilerplate code required for building |NCS| applications:
+
+         .. code-block:: console
+
+            west zephyr-export
+..
+
+If you used the default locations, your directory structure now looks similar to this:
+
+.. code-block:: none
+
+   <home>/
+   ├─── toolchains/
+   │  └─── <toolchain-installation>
+   └─── <west-workspace>/
+      ├─── .west/
+      ├─── bootloader/
+      ├─── modules/
+      ├─── nrf/
+      ├─── nrfxlib/
+      ├─── zephyr/
+      └─── ...
+
+In this simplified structure preview, *<home>* corresponds to :file:`ncs/` and *<toolchain-installation>* and *<west-workspace>* correspond to the version names you installed.
+There are also additional directories, and the structure might change over time, for example if you later :ref:`change the state of development to a different revision <updating_repos>`.
+The full set of repositories and directories is defined in the manifest file.
+
+.. _build_environment_cli:
+
+Set up the command-line build environment
+*****************************************
+
+In addition to the steps mentioned above, if you want to build and program your application from the command line, you have to set up your command-line build environment by defining the required environment variables every time you open a new command-line or terminal window.
+See :ref:`zephyr:important-build-vars` for more information about the various relevant environment variables.
+
+Define the required environment variables as follows, depending on your operating system:
+
+.. tabs::
+
+   .. tab:: Windows
+
+      Navigate to the :file:`ncs` directory and enter the following command: ``zephyr\zephyr-env.cmd``
```

Comment:
This should state using what, because someone reading this is going to open a file explorer view, go to this file, double click it and think what's that meant to do

Reply:
Rephrased.